java-api/src/main/java/com/sap/cloud/security/token/SecurityContext.java

```diff
@@ -21,6 +21,7 @@ private SecurityContext() {
 	}
 
 	private static final ThreadLocal<Token> tokenStorage = new ThreadLocal<>();
+	private static final ThreadLocal<String> planStorage = new ThreadLocal<>();
```

Comment:
As the header is now supposed to contain a list of service plans separated by comma including escaped quotes (to be confirmed), maybe we should instead use ThreadLocal<List<String>> and convert the raw String header value to a list of plans without escape quote chars inside the setter? Otherwise parsing will be required on each access to the plans and there is no documentation yet which format consumers can expect this String to be.